src/analysis/rust_type.rs

```diff
@@ -79,11 +79,15 @@ fn rust_type_full(env: &Env, type_id: library::TypeId, nullable: Nullable, by_re
         rust_type = rust_type.map_any(|s| format!("&{}", s));
     }
     if *nullable {
-        // ignore nullability of containers, they don't require wrapping in an Option
-        match TypeKind::of(&env.library, type_id) {
-            TypeKind::Pointer |
-                TypeKind::Object |
-                TypeKind::Interface => rust_type = rust_type.map_any(|s| format!("Option<{}>", s)),
+        use analysis::conversion_type::ConversionType::*;
+        match ConversionType::of(&env.library, type_id) {
+            Owned => {
+                // ignore nullability of containers, they don't require wrapping in an Option
+                match *type_ {
+                    library::Type::List(..) => (),
```

Comment:
Sorry, don't understand. I added `Type::List` as it only exception before 5962614835f7bd7c2ad016f2f4382864ea2fda00. Currently don't found any other usable containers.
I want restore `skip_option` that removed by your commit, as it simplified this condition.